src/rules/newline-after-import.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
         nextNode = nextNode.decorators[0]
       }
 
-      if (getLineDifference(node, nextNode) < 2) {
+      const options = context.options[0] || { newlines: 1 }
```

Comment:
let's add a rule schema that enforces that `newlines` is an integer? (ideally a positive one)